pages.fr/common/git-stash.md

```diff
@@ -0,0 +1,36 @@
+# git stash
+
+> Mettre les changements locaux de Git dans un endroit temporaire (Le stash).
+> Plus d'informations: <https://git-scm.com/docs/git-stash>.
+
+- Mettre les changements courants (sauf ceux provenant de fichiers qui ne sont pas suivi) dans un *stash*:
```

Comment:
Any particular reason `*stash*` is in italics here? Just curious, since this isn't the case in the original page.

I'm unsure as to whether this will be rendered correctly - I can't remember whether we've done this elsewhere or not (I don't remember any specific examples off-hand).

Reply:
Because there isn't a universal French equivalent for stash so I used the English word but since it's still in English I have to put it in italics.

Reply:
Ah, I see. This isn't technically against the client spec, so this should be ok. I'd just like another opinion  before we go ahead though, if you don't mind waiting on this PR for a little while.

Reply:
@sbrl If I'm not completely mistaking, we already do this in some pt_BR pages so I believe this is fine.

Reply:
Oh, sounds good! Just wanted to check. In that case, it's fine as-is.